src/csv_to_TM.py

```diff
@@ -0,0 +1,39 @@
+import csv
+import PYSHP
+from PYSHP import Writer
+
+with open('intervallen.csv', newline='') as csvfile:
```

Comment:
puur als voorstel: mss een optie om er eerder een functie van te maken, die je dan makkelijk kan hergebruiken:

```
def csv_to_tm(input_csv, output_gpx):
    with open(input_csv, newline='') as csvfile:
        ....
```

Zo kan je dan in andere scripts die functie inladen (`from csv_to_TM import csv_to_tm`).